api/scpca_portal/common.py

```diff
@@ -1,5 +1,7 @@
 from scpca_portal.enums import JobStates, Modalities
 
+TODO_AFTER_DATASET_RELEASE = "Remove after the dataset release"
```

Comment:
Interesting idea but let's just put this inline. The point is that we will be deleting it anyway.

Reply:
Sure!